graph_shortest_path/routing.py

```diff
@@ -42,17 +42,42 @@ def find_vertex(self, value):
         @return {Vertex} the found Vertex
         """
         # !!!! IMPLEMENT ME
-        pass
+        for vertex in self.vertices:
+            if vertex.value == value:
+                return vertex
+        return None
 
     def bfs(self, start):
         """
         Breadth-First search from an input starting Vertex
-        Should maintain parent references back from neighbors to their parent.
+        Should maintain parent references back from neighbors to their parent. 
 
         @param {Vertex} start: The starting vertex
         """
         # !!!! IMPLEMENT ME
-        pass
+        for vertex in self.vertices:
```

Comment:
We don't have to do this since the Vertices are already white by default. Check line 15.